.bazelrc

```diff
@@ -194,9 +197,9 @@ build:unix --workspace_status_command=./tools/unix/workspace-status.sh
 build:unix --cxxopt='-std=c++20' --host_cxxopt='-std=c++20'
 build:unix --@capnp-cpp//src/kj:libdl=True
 
-build:unix --action_env=BAZEL_COMPILER=clang
-build:unix --action_env=CC=clang
-build:unix --action_env=CXX=clang++
+build:unix --action_env=BAZEL_COMPILER=clang-18
```

Comment:
I assume this is unintended?
We don't want to choose clang version here but rather use the system default.
We don't require a specific clang version but rather a minimum version so setting this here will break contributors build environments

Reply:
Oh yeah woops that's supposed to be just for me